### ISSUE TITLE
Skype+Omegle: can't use/log-in because of new prompt

### ISSUE THREAD

**Issue by tealkami** [miranda-ng/miranda-ng#2947]
When you log in **web.skype.com** , it will prompt this :
```
Stay signed in?
Stay signed in so you don't have to sign in again next time.
Don't show this again
```
Which is probably saved in cookies, so check mark `Don't show this again` doesn't work for Miranda. It does ask that question everytime when you log in incognito

**Omegle** does prompt everytime this :
```
By checking the box you acknowledge that you have reviewed and agree to be bound by Omegle’s Terms of Service, Privacy Policy, and Community Guidelines.

Omegle may not be used by persons under the age of 13. Persons between 13 and 18 years of age may only use Omegle with the permission and under the supervision of their legal guardian. See our Terms of Service for more info. By checking the box you acknowledge and represent that you comply with these age restrictions.
```

Both problem are prompt/pop up related.

**Comment by georgehazan** [comment]
> so check mark Don't show this again doesn't work for Miranda
of course, it doesn't, because Miranda is not a browser and doesn't save cookies

**Comment by tealkami** [comment]
Cookies or no cookies, doesn't matter. But please fix it, I miss having Skype+Omegle in Miranda.

In both cases only needed is to add code for the prompt to work again. 
Something like.... Miranda receive the prompt from Skype, it click "no" and then continue as before
for Omegle, if Miranda receive that prompt it would automatically check both checkmarks and then click on the "Confirm & continue"

**Comment by tweimer** [comment]
The new prompt on Omegle Website has nothing to to with Miranda, Miranda uses the REST endpoint, not the website itsself. What is the problem with Omegle?

**Comment by tealkami** [comment]
tweimer: This quote does describe you perfectly: "The ability to speak does not make you intelligent."
I really don't care what code, requests, programing are called in Miranda, it use website in both cases (Skype+Omegle), and you are just arrogant prick, who just wrote some imaginary bullshit. You didn't even try to use the omegle plugin, because if you would , you would know it DOESN'T WORK, guess what, that I already wrote everything: describe what is the problem, and propose a way what to do/where to start to look.
It make me extra angry, when someone who did literally nothing ask the stupid question "what is wrong/problem?". If you really did use the website or the plugin as it should be used, you would notice it doesn't work .... 
I can guess, you just downloaded the plugin, click to go online in Miranda and that's it, you didn't go to website to see the new prompt, or use the plugin: to write /new, so it would show the error and not connect you to random person ....

I did write about that problem in Skype years ago, everytime Skype Web did had some prompt on web.skype.com, Miranda NG couldn't log in, only solution was to log in into web.skype.com, click the request it was asking, and then Miranda NG was working again. Noone in Miranda Team did add code for those prompt/request, it just didn't log in, and only way you saw some problem in Miranda was in network logs.

Well now, the prompt/request in Skype is there all the time, even if you click yes/no, it doesn't go away as before
and the same new step is on omegle website too...

**Comment by tweimer** [comment]
Why such an agressive behaviour? I did not test it, I did have other things to do. Can we please come back to a professional communication style?

I did commit a fix, please test it and tell me if it helps, because I cannot test it for now. A thank you would be appreciated in that case.
